pkg/controller/migmigration/backup.go

```diff
@@ -84,33 +66,11 @@ func (t *Task) ensureStageBackup() (*velero.Backup, error) {
 	newBackup.Labels[StageBackupLabel] = t.UID()
 	newBackup.Spec.IncludedResources = stagingResources
 	newBackup.Spec.LabelSelector = &labelSelector
-	foundBackup, err := t.getStageBackup()
+	err = client.Create(context.TODO(), newBackup)
 	if err != nil {
 		return nil, err
 	}
-	if foundBackup == nil {
```

Comment:
I think we still need this block. It makes sure we don't attempt to create the CR if it already exists. We do need to remove the !t.equalsBackup... block, though.